app/src/main/java/com/conupods/OutdoorMaps/Models/PointsOfInterest/PlacesOfInterest.java

```diff
@@ -1,46 +1,38 @@
 package com.conupods.OutdoorMaps.Models.PointsOfInterest;
 
-public class PlacesOfInterest
-{
-    private String[] html_attributions;
+public class PlacesOfInterest {
+    private String[] htmlAttributions;
 
     private Place[] results;
 
     private String status;
 
-    public String[] getHtml_attributions ()
-    {
-        return html_attributions;
+    public String[] gethtmlAttributions() {
+        return htmlAttributions;
     }
 
-    public void setHtml_attributions (String[] html_attributions)
-    {
-        this.html_attributions = html_attributions;
+    public void setHtmlAttributions(String[] htmlAttributions) {
+        this.htmlAttributions = htmlAttributions;
```

Comment:
Do not change the name of this attribute, it must match the attribute name that is within the JSON response object. Revert change.